File: web/backend/app/schemas/forward.py
from uuid import UUID
from pydantic import BaseModel

class ForwardRequest(BaseModel):
    phone: str
    agent_id: UUID
    recipient_user_ids: list[UUID]
    mode: str = "fyi"  # "fyi" or "action"
    note: str | None = None

class ForwardResponse(BaseModel):
    thread_id: UUID

File: web/frontend/src/api/forward.ts
import { api } from './client';

interface ForwardRequest {
  phone: string;
  agent_id: string;
  recipient_user_ids: string[];
  mode: 'fyi' | 'action';
  note?: string;
}

export const forwardApi = {
  forward: (req: ForwardRequest) => api.post<{ thread_id: string }>('/forward', req),
};
